### ISSUE TITLE
Feature Request: Shortcut for inserting a TODO 

### ISSUE THREAD

**Issue by preslavrachev** [drgrib/alfred-bear#20]
I use the `bn` shortcut quite often, many times for adding simple TODOs or reminders. Although adding a new note with just a title is more than fine within Bear, it requires a bit of cleaning up every now and then. 

I'd propose the adding of an additional shortcut: e.g. `bt`, which would either:
- pre/append a new TODO to a single note defined in the workflow's settings (default: TODOs)
- or, create a new note without a title, but only with the TODO text on the first line (Bear is quite OK with this). Each note may have an associated hashtag (#TODO by default). Optionally, when adding a TODO, the app can skip opening it up, since it is very rare that one needs to add more details right away.

<img width="323" alt="image" src="https://user-images.githubusercontent.com/86060/78771823-13586380-7991-11ea-859e-3d64215cd96c.png">

@drgrib what do you think?

**Comment by preslavrachev** [comment]
As always, I'm OK with helping to code this feature, if everyone agrees that it would be useful

**Comment by nickwild-999** [comment]
@preslavrachev yes this would be really useful. A todo (and appending to an existing bulleted list) is what I was referring to in #19

**Comment by drgrib** [comment]
I'm sorry to be a damper on this guys but it feels a little messy and I'd rather not complicate the code unless we have a compelling, clear, clean, in-scope feature to add

Also, while you are both requesting a similar feature (appending items to notes), they are actually different. #19 is trying to add _bullets_ to arbitrary selected notes by title and this is trying to add _todos_ to a single note predefined in a settings file or an untitled note.

A settings file doesn't currently exist and would be a new system added to the workflow just for this one feature.

It all just doesn't seem to fit into the scope of this workflow, which is focused on searching, accessing, and creating notes with specific titles and tags. If you'd like to append items to a static or untitled note, I think it would probably make sense to create a new, different workflow dedicated to that task. I'd be happy to have you post a link to it in this issue.

**Comment by drgrib** [comment]
Also, in the spirit of open source, please feel free to reference or reuse code from this repo and/or create a generic library based on its current access functions if you do decide to create a new workflow.
